src/vivarium_nih_us_cvd/components/risk_correlation.py

```diff
@@ -47,6 +52,38 @@ def setup(self, builder: Builder) -> None:
         self.input_draw = builder.configuration.input_data.input_draw_number
         self.random_seed = builder.configuration.randomness.random_seed
         self.correlation_data = pd.read_csv(paths.FILEPATHS.RISK_CORRELATION)
+        self.population_attributable_fractions = (
+            self.get_population_attributable_fraction_source(builder)
+        )
+        self.register_paf_modifiers(builder)
+
+    #################
+    # Setup methods #
+    #################
+
+    def get_population_attributable_fraction_source(
+        self, builder: Builder
+    ) -> Dict[str, LookupTable]:
+        paf_data = builder.data.load(
+            "risk_factor.joint_mediated_risks.population_attributable_fraction"
+        )
+        pafs = {}
+        for name, group in paf_data.groupby(["affected_entity", "affected_measure"]):

```

Comment:
If you think it's simpler/cleaner, you can also loop through the groupby using 
`for (name_0, name_1), group in paf_data.groupby()`

Reply:
ooh yeah I like that more